src/api/app/components/bs_request_comment_component.rb

```diff
@@ -16,7 +16,7 @@ def initialize(comment:, commentable:, level:, diff: nil, show_username: true)
   end
 
   def range
-    line_index = @comment.diff_ref.match(/diff_[0-9]+_n([0-9]+)/).captures.first
-    ((line_index.to_i - 4).clamp(0..)..(line_index.to_i - 1))
+    line_index = @comment.diff_line_number
+    ((line_index - 4).clamp(0..)..(line_index - 1))
```

Comment:
💭 a lot of magic numbers in this one...

Reply:
Yes. :sweat_smile:

Reply:
`4` refers to the number of lines we display at most in a diff comment. It should be a var to make this clearer